includes/rest/class-nodeinfo.php

```diff
@@ -173,7 +173,7 @@ public static function discovery( $request ) {
 		$discovery['links'] = array(
 			array(
 				'rel' => 'http://nodeinfo.diaspora.software/ns/schema/2.0',
-				'href' => \get_rest_url( null, 'activitypub/1.0/nodeinfo' ),
+				'href' => \get_rest_url( null, ACTIVITYPUB_REST_NAMESPACE . '/nodeinfo' ),
```

Comment:
I would also use `sprintf` here to be consistent.

Reply:
I think that, instead, I'm going to make an internal helper for this.